charm/kafka/templates/server.properties

```diff
@@ -65,7 +65,11 @@ socket.request.max.bytes=104857600
 ############################# Log Basics #############################
 
 # A comma separated list of directories under which to store log files
+{% if log_dirs is defined %}
```

Comment:
I think this should always be defined. Since we need to know the log_dir to get_or_create_broker_id, it should be defined by the time we render the template.